pom.xml

```diff
@@ -729,7 +730,7 @@
                 </requirePluginVersions>
                 -->
                 <requireJavaVersion>
-                  <version>[11,]</version>
+                  <version>[{javaTarget},]</version>
```

Comment:
```suggestion
                  <version>[${javaTarget},]</version>
```
?

Reply:
well this is interesting...

https://github.com/jenkinsci/trilead-ssh2/pull/107 did not blow up in anyway shape or form...  
I guess `8` > `{javaTarget}`